plugins/wallabag.koplugin/main.lua

```diff
@@ -181,6 +185,21 @@ function Wallabag:addToMainMenu(menu_items)
                             self:setFilterTag(touchmenu_instance)
                         end,
                     },
+                    {
+                        text_func = function()
+                            local tags
+                            if not self.ignore_tags or self.ignore_tags == "" then
+                                tags = _("None")
```

Comment:
```suggestion
                                return _("Ignore tags")
```

I don't think anything more is required, but if you think something like `none` is a good addition it should just be part of the full string instead of harder to localize through programmatic cleverness. ;-)

Reply:
I'm not sure to understand the issue, so I remove the "None" string, it will just be empty now.